docs/docs/20-usage/51-plugins/10-plugins.md

```diff
@@ -30,6 +30,14 @@ pipeline:
 
 Plugins are just pipeline steps. They share the build workspace, mounted as a volume, and therefore have access to your source tree.
 
+## Finding Plugins
+
+For existing plugins there's a few places you can look:
+
+- [Official Woodpecker Plugins](https://woodpecker-ci.org/plugins)
+- [Drone Plugins](http://plugins.drone.io)
+- [The Geek Lab Drone Plugins](https://drone-plugin-index.geekdocs.de/plugins/drone-matrix/)
```

Comment:
What do you think about only having the link to the official woodpecker plugin list here and adding the other two links on the plugins list page somewhere like: `There are also other plugin lists with additional plugins. Keep in mind that [Drone](https://www.drone.io/) plugins are generally supported, but could need some adjustments and tweaking.`

Reply:
Refactored a bit and added a Tip section

<img width="1111" alt="CleanShot 2023-03-05 at 20 25 18@2x" src="https://user-images.githubusercontent.com/64056131/223025962-eb0254bc-5d19-4627-b413-8b1cdf332c7b.png">

Let me know what you think

Now re-reading what you wrote I realize that isn't what you meant 🤦

Reply:
@6543 This wasn't ready yet. 👆

Reply:
Hmm i did go throu your points and thought they where addressed. The wording also looked good 🤔

Reply:
is it only a smal missing bit?

Reply:
That's why I haven't resolved it yet 😉 

The current place is just confusing / could be missed, so the idea was to link to https://woodpecker-ci.org/plugins and place a hint about other indexes there.

Reply:
Ok, i still dont get the difference to now 100%.

@hay-kot if you do, just send a pull i guess else I'll DM you and create a patch @anbraten 

Sorry for the extra work